### PR TITLE
JWWEB-731 Update import wallet

### PR DESCRIPTION
**Pull request by IViol**

### CODE REVIEW

File: src/components/TitleHeader/TitleHeader.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+|}
+
+export function TitleHeader({
+  onBack: handleClick,
+  title,
+}: Props) {
+  return (
+    <div className={titleHeaderStyle.core}>
+      <h1 className={titleHeaderStyle.title}>{title}</h1>
+      {handleClick && (
```
Comment (stcherenkov) [inline src/components/TitleHeader/TitleHeader.js]: JLinkBack?

Comment (stcherenkov) [reply src/components/TitleHeader/TitleHeader.js]: Nope, I'm wrong

File: src/layouts/StartLayout/StartLayout.js [hunk 2]
```diff
@@ -9,23 +9,27 @@ import svgLogoBlue from 'public/assets/logo/logo-blue.svg'
 import startLayoutStyle from './startLayout.m.scss'
 
 type Props = {|
-  className?: ?string,
   +children: React$Node,
+  +className: ?string,
+  +hasNotLogo: boolean,
```
Comment (stcherenkov) [inline src/layouts/StartLayout/StartLayout.js]: `hasNoLogo` sounds better, but it's highly opinionated

File: src/styles/_base.scss [hunk 3]
```diff
@@ -113,7 +113,7 @@ $default-animation: cubic-bezier(0, 0, 0.2, 1);
  * Shadows
  */
 $blue-shadow: 0 3px 6px rgba(0, 61, 198, 0.2);
-$blue-shadow-2: 0 6px 12px rgba(0, 61, 198, 0.4);
-$normal-shadow: 0 3px 6px $main-black-5;
-$normal-shadow-2: 0 6px 12px rgba(8, 30, 57, 0.1);
-$select-shadow: 0 7px 18px $main-black-10;
+$blue-shadow-2: 0 9px 18px rgba(0, 61, 198, 0.2);
```
Comment (stcherenkov) [inline src/styles/_base.scss]: Did you check all the places we use those shadows?

Comment (IViol) [reply src/styles/_base.scss]: yeah

**Pull request merged**